README.md

```diff
@@ -454,3 +454,16 @@ You can use one of the following solutions:
     `~/Library/Group\ Containers/group.com.docker/settings.json` file and
     add `/opt` to `filesharingDirectories`.
 
+
```

Comment:
Let's add a section named 

`Configure private PyPI repository`
```
To allow installation of PyPI packages from private repositories define PIP_EXTRA_INDEX_URL environment variable that should contain an [alternate package index location](https://mothergeo-py.readthedocs.io/en/latest/development/how-to/alternate-pypi.html#how-to-use-an-alternate-pypi-package-index), eg.

PIP_EXTRA_INDEX_URL=http://pypi.mydomain.com:8080
```
after section `### Install or remove PyPI packages`

Reply:
[Addressed in this commit](https://github.com/GoogleCloudPlatform/composer-local-dev/pull/21/commits/6e3c453508299a1d3980cceea0cd170db33891fa)